IFC4x3/PropertyConstants/e/EXTERNAL_3gdw0DUw5Dnhn8UTc5nQnZ.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="EXTERNAL_3gdw0DUw5Dnhn8UTc5nQnZ" Name="EXTERNAL" UniqueId="ea9fa00d-7ba1-4dc6-bc48-79d985c5ac63" />
```

Comment:
There are a couple of externals already. Would we consider re-using?

Reply:
+1